src/controller/clients/shape-search-client.js

```diff
@@ -52,11 +52,13 @@ class ShapeSearchClient extends BaseClient {
     for (let unit in units) {
       // console.log("Unit: ", units[unit])
       for (let i in mObj) { mObj[i] = ""  }
-      let mObjPush = JSON.parse(JSON.stringify(mObj))
-      mObjPush.location = units[unit][0].location
-      mObjPush.condo = true
-      mObjPush.pwd_parcel_id = units[unit][0].pwd_parcel_id
-      data.rows.push(mObjPush)
+      let mObjPush = JSON.parse(JSON.stringify(mObj));
```

Comment:
Can we find a better name than `mObjPush`? I don't really know what that means...

Reply:
Also, what's the purpose of this line and the line above?

Reply:
The other search method (geocode) returns a record for the condo building and parcel. Shape search responses from Carto don't have this record so it is created by copying one of the records and pushed into all of the other non-condo results being displayed.